src/OpenApiService.php

```diff
@@ -44,7 +44,11 @@ public function createSchema()
 		foreach ($this->getEndpoints() as $endpoint) {
 			$endpointId++;
 
-			$pathItem = new PathItem();
+			$pathItem = $paths->getPath($endpoint->getMask());
+			if ($pathItem === NULL) {
```

Comment:
Is this path used anywhere?

Reply:
What do you mean?

Path (from `paths`) is filled by `$endpoint->getMask()`. If you create a method for each `GET`|`POST`|... using the same mask, then only last endpoint is exported using `OpenApiService`.

Reply:
Sorry, I ment, is variable `$pathItem` used anywhere? I don't see it.

Reply:
Yes, [here](https://github.com/Scalesoft/openapi/blob/1aac536a32fdf793ee46d9f48ce79b124ee736f8/src/OpenApiService.php#L129)

Reply:
Oh, thanks.